src/components/calcite-radio-group-item/calcite-radio-group-item.scss

```diff
@@ -1,101 +1,93 @@
 :host {
-  display: flex;
-  align-self: stretch;
-  cursor: pointer;
+  @apply flex self-stretch cursor-pointer;
   transition: background-color 0.1s ease-in-out, border-color 0.1s ease-in-out;
 }
 
 :host label {
-  display: flex;
+  @apply flex 
+    text-color-3 
+    m-1 
+    box-border 
+    pointer-events-none 
+    items-center;
   flex: 1;
-  color: var(--calcite-ui-text-3);
   border: 1px solid transparent;
-  margin: 4px;
-  box-sizing: border-box;
   transition: background-color 0.1s ease-in-out, border-color 0.1s ease-in-out;
-  pointer-events: none;
-  display: flex;
-  align-items: center;
 }
 
 :host([layout="horizontal"]) label {
-  justify-content: center;
+  @apply content-center;
 }
 
 // focus styles
 :host {
-  @include focus-style-base();
+  @apply focus-base;
 }
 :host(:focus) {
-  @include focus-style-inset();
+  @apply focus-inset;
 }
 
 :host([scale="s"]) label {
-  @apply text--2;
-  padding: 0.25rem 0.75rem;
+  @apply text--2 px-2;
+  padding-top: 0.125rem;
+  padding-bottom: 0.125rem;
 }
 
 :host([scale="m"]) label {
-  @apply text--1;
-  padding: 0.4rem 1rem;
+  @apply text--1 px-3;
+  padding-top: .375rem;
+  padding-bottom: .375rem;
```

Comment:
It would be ideal if we could remove these padding values.

Reply:
I think we can revisit this component later, to get the height to S24px, M32, L48, this is what Bryan and I worked with.

Reply:
Should we create a new issue for this or add it to #1500?

Reply:
We should create an issue for this, we might need to rethink the design? CC'ing @bstifle